pushy/src/main/java/com/relayrides/pushy/apns/ApnsClient.java

```diff
@@ -469,7 +471,7 @@ public void operationComplete(final ChannelFuture future) throws Exception {
                                 if (ApnsClient.this.reconnectionPromise != null) {
                                     log.debug("Disconnected. Next automatic reconnection attempt in {} seconds.", ApnsClient.this.reconnectDelaySeconds);
 
-                                    future.channel().eventLoop().schedule(new Runnable() {
+                                    scheduledReconnectFuture = future.channel().eventLoop().schedule(new Runnable() {
```

Comment:
Minor to the point that I almost feel bad asking: could you please qualify this with `ApnsClient.this`?